nupic/embodied/envs/wrappers.py

```diff
@@ -42,6 +42,46 @@ def unwrap(env):
         return env
 
 
+class LazyFrames(object):
+    def __init__(self, frames):
+        """
+        from https://github.com/openai/baselines/blob/ea25b9e8b234e6ee1bca43083f8f3cf974
+        143998/baselines/common/atari_wrappers.py#L229
+
+        This object ensures that common frames between the observations are only stored once.
+        It exists purely to optimize memory usage which can be huge for DQN's 1M frames replay
+        buffers.
+        This object should only be converted to numpy array before being passed to the model.
+        You'd not believe how complex the previous solution was."""
```

Comment:
Hahaha interesting comment on how complex the previous solution was - does this replaces one existing wrappers we had, or is it new?

Reply:
Previously we used from baselines.common.atari_wrappers import FrameStack. Now with the switch to stable-baselines we had just added the FrameStack function but it still required the LazyFrames function to work so I added it there but it is from https://github.com/openai/baselines/blob/ea25b9e8b234e6ee1bca43083f8f3cf974143998/baselines/common/atari_wrappers.py We can maybe replace both with  VecFrameStack from stable_baselines3, I'll have to check if they do the same thing.